docker/dev.yml

```diff
@@ -0,0 +1,55 @@
+
+version: "3.7"
+
+services:
+  db:
+    image: rethinkdb:latest
+    restart: unless-stopped
+    ports:
+      - "8080:8080"
+      - "29015:29015"
+      - "28015:28015"
+    volumes:
+      - rethink-data:/data
+    networks:
+      - parabol-network
+  postgres:
+    image: postgres:12.5
+    restart: unless-stopped
+    env_file: ../.env
+    ports:
+      - "5432:5432"
+    volumes:
+      - "../packages/server/postgres/postgres.conf:/usr/local/etc/postgres/postgres.conf"
```

Comment:
@tiffanyhan i wanted to get rid of that DOCKERFILE so a fun little trick is to mount config files as volumes. doing this means we don't have to copy it into the container